### PR TITLE
[added new article] A Comprehensive Guide to Comparing Hardware and Software Wallets on Celo

### PR DESCRIPTION
**Pull request by boyfem**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @phenzic

### CODE REVIEW

File: blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+authors:
+  - name: ✍️ Boyejo Oluwafemi
+url: https://github.com/boyfem
+image_url: https://github.com/boyfem
+tags: [beginner]
+hide_table_of_contents: true
+slug: /a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo
+---
+
+# A Comprehensive Guide to Comparing Hardware and Software Wallets on Celo
```
Comment (Julius170) [inline blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md:13]: you can remove the topic in this tutorial file

File: blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+# Introduction​
```
Comment (Julius170) [inline blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md:15]: add another hashtag here

File: blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+Over the past decade, cryptocurrencies have evolved from an undervalued asset to an exceedingly popular investment. Today, cryptocurrencies make up a significant part of the average trader's investment portfolio. Therefore, a wallet is a necessity if you must purchase, trade, or use cryptocurrencies in any other way. This article will compare hardware and software wallets in detail, highlighting the key differences between the two wallet types and their different advantages and disadvantages. 
+
+# Prerequisites​
```
Comment (Julius170) [inline blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md:19]: add another hashtag here

File: blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+Knowing what crypto wallets are and their applications and possibilities on the Celo blockchain will come in handy for this article. Check out the link below.
+https://docs.celo.org/blog/tutorials/crypto-wallets-the-celo-way
+
+# What is a Wallet?
```
Comment (Julius170) [inline blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md:23]: add another hashtag here

File: blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+- Software wallets are more convenient for everyday use since they can be accessed from any location with an internet connection.
+- They often accept a wider range of cryptocurrencies.
+- They frequently have security features like multi-sig support, seed phrase recovery, and two-factor authentication.
+
+### Cons of Software Wallets 
+- Software wallets keep the private keys on the user's device or in the cloud, making them more susceptible to hacking attempts if the user's device or the cloud server are both hacked.
+- The user may permanently lose access to the assets if they lose their smartphone or forget their password.
+- They could be impacted by viruses, malware, and other cyberattacks. 
+
+# Conclusion​
```
Comment (Julius170) [inline blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md:104]: add another hashtag here and a space after the header

File: blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+In summary, when it comes to storing cryptocurrency, both hardware and software wallets have their pros and cons. Hardware wallets provide a more secure form of storage, while software wallets are more convenient for everyday use. The choice between them will depend on an individual's specific needs and preferences and the supported coins that determine the type of wallet.
+# About the Author​
```
Comment (Julius170) [inline blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md:106]: add another hashtag here and a space after the header

File: blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+Boyejo Oluwafemi is a hardware product developer working at the intersection of hardware and blockchain technology. He’s working to leverage his wealth of experience working on several products ranging from smart health devices to sporting gadgets to deliver smart payment solutions for crypto for a more inclusive future. You can follow me on Twitter using this [link](https://twitter.com/livingcisco)
+# References​
```
Comment (Julius170) [inline blog/02-17-23-a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo/a-comprehensive-guide-to-comparing-hardware-and-software-wallets-on-celo.md:108]: add another hashtag here and a space after the header

**Review by Julius170 — COMMENTED** [review]
Awesome 
you can now make changes based on the review @boyfem

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**